web-client/iron-remote-gui/src/interfaces/UserInteraction.ts

```diff
@@ -31,4 +32,6 @@ export interface UserInteraction {
     setCursorStyleOverride(style: string | null): void;
 
     onSessionEvent(callback: (event: SessionEvent) => void): void;
+
+    resizeDynamic(width: number, height: number, scale?: number): void;
```

Comment:
naming: Why "Dymanic"? What about naming it `resize`?

e.g.: `session.resize(width, height)`

Reply:
will update for public API